app/services/sufia/admin_set_service.rb

```diff
@@ -21,15 +21,32 @@ def search_results_with_work_count(access)
       end
     end
 
+    # Return AdminSet selectbox options based on access type
     # @param [Symbol] access :read or :edit
     def select_options(access = :read)
       search_results(access).map do |element|
-        permission_template = PermissionTemplate.find_by(admin_set_id: element.id)
-        visibility = permission_template.visibility if permission_template
-        # Add HTML5 'data' attributes corresponding to permission template fields
-        # Used to limit visibility options of new works (via JS) when an AdminSet is selected
-        [element.to_s, element.id, { 'data-visibility' => visibility }]
+        [element.to_s, element.id, data_attributes(element)]
```

Comment:
Good point. This is a vestige of someone else's code, but updating that name would clarify this method. Will refactor